operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -185,16 +192,16 @@ The [Technical Writing Jira board](https://mattermost.atlassian.net/secure/Rapid
 If you have a feature documentation requirement, please create a ticket and include the following fields:
 
 * Label: Documentation
-* Team: Tech Writing 
-* Assignee: Justine Geffen
+* Team: Tech Writing
+* Assignee: Unassigned
 
 Depending on what is required, please also include a sprint as well as a fix version.
 
 If your request is not related to feature documentation, include the following fields:
 
 * Label: Documentation
-* Team: Tech Writing 
-* Assignee: 
+* Team: Tech Writing
+* Assignee:
```

Comment:
Fill in or remove

Reply:
Removed as I need to run the process past the documentation working group.